### DISCUSSION TITLE
Projects Beta: The ID in table view is very confusing

### DISCUSSION THREAD

**Discussion by Akuka** [community/community#9354] [Projects and Issues] [answered]
The ID in table view is very confusing as its not the issue ID, which actually not displayed at all in the table view

**Comment by fdesmottes** [comment]
A similar issue : #8491

**Comment by rileybroughten** [comment, marked as answer]
Making this as a duplicate of [#5904](https://github.com/community/community/discussions/5904), keep an eye on the original discussion post for updates! 👀
